dask_glm/tests/test_estimators.py

```diff
@@ -64,7 +64,7 @@ def test_lm(fit_intercept):
 @pytest.mark.parametrize('fit_intercept', [True, False])
 def test_big(fit_intercept):
     import dask
-    dask.set_options(get=dask.get)
+    dask.config.set(scheduler='synchronous')
```

Comment:
Hmm, while we're at it could you update these to use a context manager?

Reply:
For sure, good idea

Reply:
Also, should I move `import dask` outside the tests? There's probably a reason for it being inside the test, but it's not clear to me